docs/sphinx/manual/building_hpx.rst

```diff
@@ -71,7 +71,7 @@ Quick start
 
 Here, you will use the command-line, non-interactive CMake interface.
 
-#. Download and install CMake here: |cmake_download|_. Version 3.13 is the
+#. Download and install CMake here: |cmake_download|_. Version 3.17 is the
```

Comment:
There are a couple of tables in this file as well which still say 3.13. Would you mind updating those?